hydradx/model/amm/omnipool_amm.py

```diff
@@ -1344,23 +1342,6 @@ def weight_i(state: OmnipoolState, i: str) -> float:
     return state.lrna[i] / state.lrna_total
 
 
-def simulate_swap_lrna(
```

Comment:
Why is this no longer necessary?

Reply:
It's not necessary because you can use simulate_swap(tkn_buy/sell = 'LRNA') to the same effect, and doing so lets us put the oracle update code all in one place.